charts/opentelemetry-operator/values.yaml

```
@@ -153,6 +153,10 @@ admissionWebhooks:
   failurePolicy: Fail
   secretName: ""
 
+  ## Mutating webhooks are called sequentially. In some cases you would like your MutatingWebhooks
+  ## to run in specific order. In such case, add a prefix to all of them to sort alphabetically.
```

```suggestion
  ## Adds a prefix to the mutating webook name.
  ## This can be used to order this mutating webhook with all your cluster's mutating webhooks.
```

My suggestion is to call out that the ordering is necessary not because this chart has multiple mutating webhooks but because your cluster might.

---

thanks